XenCenterLib/Win32.cs

```diff
@@ -167,7 +167,7 @@ public static void StopFlashing(IntPtr hwnd)
         public const UInt32 ERROR_DISK_FULL = 112;
 
         public const int CBN_CLOSEUP = 8;
-        public const int EN_KILLFOCUS = 512;
+        public const int EN_STOPFOCUS = 512;

```

Comment:
I think we should keep this as it was because it's Windows APIs. Although it's the value rather than the name that matters, keeping the same name makes it easier to understand what it refers to.